market_maker_keeper/erisx_market_maker_keeper.py

```diff
@@ -376,5 +379,49 @@ def place_order_function(new_order_to_be_placed):
             else:
                 logging.info(f"New {side} Order below size minimum of {min_amount}. Order of amount {amount} ignored.")
 
+    def check_cancellations(self):
+        """
+        Check to see if ErisX has cancelled any of our orders.
+        If an order has been cancelled, remove it from the order book
+        """
+        with self._lock:
```

Comment:
I only see `_lock` used in one place.  Is `synchronize_orders` called from multiple threads simultaneously?

Reply:
Nope, that was vestigial code. I will remove it.